src/app/components/core/scheduler/list-schedules/list-schedules.component.ts

```diff
@@ -16,17 +18,18 @@ enum weekDays {
   Sat = 6,
   Sun = 7
 }
+
 @Component({
   selector: 'app-list-schedules',
   templateUrl: './list-schedules.component.html',
   styleUrls: ['./list-schedules.component.css']
 })
+
 export class ListSchedulesComponent implements OnInit {
   public scheduleData = [];
   public scheduleProcess = [];
   public scheduleType = [];
   public days = [];
-  public scheduler_name: string;
```

Comment:
@mshariq-nerd is this being used for enable/disable or edit/update? 
/cc @monika-Dianomic